librz/core/cmd/cmd_api.c

```diff
@@ -31,6 +31,24 @@
 #define MIN_SUMMARY_WIDTH    6
 #define MAX_RIGHT_ALIGHNMENT 20
 
+// Based on Config and Line order choses the vertical line shape
+static inline const char *get_vertical_line(size_t idx, size_t len, bool scr_curvy, bool scr_utf8) {
+	if (scr_curvy && !idx) {
+		return RUNE_CURVE_CORNER_TL;
+	} else if (scr_curvy && (idx == len - 1)) {
+		return RUNE_CURVE_CORNER_BL;
+	} else if (scr_utf8) {
+		return RUNE_LINE_VERT;
+	} else {
+		return "| ";
+	}
+}
+
+// return true if core and core->config aren't NULL, else false
+static inline bool is_core_and_config(RzCore *core) {
+	return (core && core->config);
```

Comment:
```suggestion
static inline bool core_config_get_b(RzCore *core, const char *option) {
	return core && core->config ? rz_config_get_b(core->config, option) : false;
```

This is what i meant.